pkg/storage/store.go

```diff
@@ -593,6 +591,13 @@ type StoreConfig struct {
 
 	// MetricsSampleInterval is (server.Context).MetricsSampleInterval
 	MetricsSampleInterval time.Duration
+
+	// EnableCoalescedHeartbeats sets whether heartbeats are coalesced. If it
+	// has not been proactively set to true when calling NewStore() (i.e.
+	// through using a config from TestStoreConfig()) it will be set off of the
+	// COCKROACH_ENABLE_COALESCED_HEARBEATS environment variable. By default it
+	// is set to false.
```

Comment:
That comment is likely to rot. Shouldn't make assumptions about what this is going to be set to in this location.